lib/rubocop/cop/style/empty_lines_around_begin_body.rb

```diff
@@ -0,0 +1,42 @@
+
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cops checks if empty lines exist around the bodies of begin-end
+      # blocks.
+      #
+      # @example
+      #
+      #   # good
+      #
+      #   begin
+      #     ...
+      #   end
+      #
+      #   # bad
+      #
+      #   begin
+      #
+      #     ...
+      #
+      #   end
+      class EmptyLinesAroundBeginBody < Cop
+        include EmptyLinesAroundBody
+
+        KIND = 'begin'.freeze
```

Comment:
Why are you confusing? I think it is good to keep constant assignment.

Note:  The `begin` keyword should be surrounded by backtick(I'll fix it).